src/classification/predict.py

```diff
@@ -62,27 +51,56 @@ def predict(x, model, patch_sz=160, n_classes=2):
         prediction[x0:x1, y0:y1, :] = patches_predict[k, :, :, :]
     return prediction[:img_height, :img_width, :]
 
+
 # generating sliding window
 def sliding_window(img, stepSize, windowSize):
     for y in range(0, img.shape[0], stepSize):
         for x in range(0, img.shape[1], stepSize):
-            yield (x, y, img[y:y + windowSize, x:x + windowSize,:])
-                        
+            yield (x, y, img[y:y + windowSize, x:x + windowSize, :])
+
+
 def main():
-    outPath = "data/WV_predicted"
-    i=0
-    for(x,y, window) in sliding_window(img, stepSize, windowSize):
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--input', type=str,
+                        help='Path and Filename of the 3-Band Multipage WV \
+                              Image', required=True)
+    parser.add_argument('-o', '--output_folder', type=str, default='./'
+                        help='Path where output will be stored.')
+
+    args = parser.parse_args()
+    model = get_model()
+    model.load_weights(weights_path)
+    head, tail = os.path.split(args.input)
+    getName = tail.split('-multipage.tif')
+    outPath = args.output_folder + "data/predicted_tiles/" + getName[0]

```

Comment:
It is needed because we do not want to hardcode where intermediate results are stored. I would argue that even `data/predicted_tiles/` can be removed.